kkl15.qmd

```diff
@@ -310,16 +300,34 @@ The plot of quantiles of model residuals over corresponding quantiles of the nor
 #| code-fold: true
 #| label: fig-qqnormm1
 #| fig-cap: Quantile-quantile plot of the residuals for model m1 versus a standard normal
-#| eval: false
-qqnorm(m1; qqline=:none)
+let ppts = inv(1024):inv(512):1
```

Comment:
is this just because of the Makie bug with xlims? We can also work around that by adding Makie as a dep and setting the compat bound to `"=0.19.6"`

Reply:
I did this as a smoothing operation and to have the plot render faster - choosing a fixed number of points equally spaces on the probability scale and evaluate both sets of quantiles.

It may not be necessary to do this to reduce the number of points that are plotted.  It may be sufficient to change to "png" format from "svg".

Reply:
@palday @kliegl I am fine with trying to go back to a call to `qqnorm` if desired.  I just found that it was taking a very long time on this example because of the large number of observations.

Reply:
I'm not opposed to doing a faster plot -- do what you think works well.